files/www/cgi-bin/status

```diff
@@ -78,6 +78,20 @@ alert_banner();
 print "<h1><big>$node";
 print " / $tactical" if $tactical;
 print "</big></h1>";
+if(-f "/etc/latlon") {
+    $rcgood=open(FILE, "/etc/latlon");
```

Comment:
move this logic before the output starts generating and save to variable.  Reference it here.